upgrade/pack_builder/BinaryObject.cpp

```diff
@@ -2,6 +2,15 @@
 #include "infra/stream/StdStringInputStream.hpp"
 #include <algorithm>
 
+namespace
+{
+    bool RemainderHasNoData(const infra::StdStringInputStream& stream)
```

Comment:
The name of this function talks about remainder and data, but the implementation is of a much lower level. I also think the name says the inverse of what it actually returns. Please consider remaining to something like StreamHasNonSpaces

Reply:
oops, thanks :)